insights/parsers/readlink_openshift_certs.py

```diff
@@ -0,0 +1,60 @@
+"""
+Commands: ReadLinkEKubeletClientCurrent and ReadLinkEKubeletServerCurrent
```

Comment:
Please update as per https://github.com/RedHatInsights/insights-core/wiki#checklist-for-reviewers